src/style.css

```diff
@@ -1 +1,100 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+input {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). In this case, you have added a top margin to the input. Ensure that you use margins consistently throughout your CSS to avoid potential margin collapse.